configs/missiles/flamer.model.cfg

```diff
@@ -1,9 +1,10 @@
 // display
 sound              models/weapons/flamer/fireloop
 
-dlight             200
-dlightColor        0.25 0.1 0.0
-dlightIntensity    0 // TODO: Find out if a value != 0 is necessary
+// This doesn't seem to do anything
```

Comment:
Then just remove it? Why keep zombie code?

Reply:
What do you mean by doesn't do anything? Like the dlight param is broken?

Reply:
I had left it to allow for further investigation.

Removed.